perfkitbenchmarker/providers/aws/aws_disk.py

```diff
@@ -33,12 +33,49 @@
 VOLUME_EXISTS_STATUSES = frozenset(['creating', 'available', 'in-use', 'error'])
 VOLUME_DELETED_STATUSES = frozenset(['deleting', 'deleted'])
 VOLUME_KNOWN_STATUSES = VOLUME_EXISTS_STATUSES | VOLUME_DELETED_STATUSES
+
+STANDARD = 'standard'
+GP2 = 'gp2'
+IO1 = 'io1'
+
 DISK_TYPE = {
-    disk.STANDARD: 'standard',
-    disk.REMOTE_SSD: 'gp2',
-    disk.PIOPS: 'io1'
+    # no need to translate 'standard' because the old and new names
```

Comment:
Understood. Quick question then: Today can a benchmark spec create a scratch disk without specifying the scratch_disk_type or equivalent in their benchmark spec? If so, we need to continue to support that behavior.

Going back to this particular line, I think we need to put back the disk.STANDARD entry here in the mapping. The goal of this mapping is to map those disk type constants as defined in the base class (disk.py) to provider-specific literal strings that can be passed to provider CLI tool to create disk. It's merely a "happy coincidence" that the disk.STANDARD constant is the same as the literal string "standard" accepted by AWS and we can use as is here. However, it'd be semantically much cleaner if we still maintain the mapping here so we don't break the layers of abstractions and make this class make assumptions about some constants defined by some other class. Doing the mapping here will also avoid future maintenance problems: e.g., if someone decides to change the disk.STANDARD  constant in disk.py in the future to some other literal string, they would have no idea to come to this class to add a mapping here.